### PR TITLE
Disable track 1 Key Vault tests that do not work live

### PR DESCRIPTION
**Pull request by heaths**
Disable some track 1 tests until #8420 is resolved.

### CODE REVIEW

**Comment by heaths** [comment]
I will not be checking in the change to eng\services.proj. I just want to avoid running the *long* tests I know work.

**Comment by heaths** [comment]
It's not obvious from the checks, but I did run these through "net - keyvault -tests" to make sure they worked as expected.

File: sdk/keyvault/Microsoft.Azure.KeyVault/tests/KeyVaultOperationsTest.cs [hunk 1]
```diff
@@ -20,6 +20,10 @@
 using Microsoft.Rest.Serialization;
 using Xunit;
 
+// Avoid compiling the test class. Disabling just the test methods still causes the fixture to instantiate and dispose, which still breaks live tests.
+// See https://github.com/Azure/azure-sdk-for-net/pull/8423 for status and more information about why these tests are disabled.
```
Comment (pakrym) [inline sdk/keyvault/Microsoft.Azure.KeyVault/tests/KeyVaultOperationsTest.cs]: I would reference the issue here not the PR

**Comment by pakrym** [comment]
Are we sure that the docs https://nunit.org/docs/2.5/ignore.html are wrong and ignoring class doesn't work? Did we file an issue on NUnit?

File: sdk/keyvault/Microsoft.Azure.KeyVault/tests/KeyVaultOperationsTest.cs (hunk 1 above)
Comment (heaths) [reply sdk/keyvault/Microsoft.Azure.KeyVault/tests/KeyVaultOperationsTest.cs]: Oops. Definitely intended the issue. I'll fix that.

**Comment by heaths** [comment]
Hmm, didn't know about the attribute. I searched for things like "nunit ignore class" and that never came up - just references to issues resolved "Won't Fix" and workarounds. I'll try that. If it works, it indeed would be easier to maintain.

**Comment by heaths** [comment]
Sorry, just remembered: these are xUnit tests, which doesn't have such a thing.

**Review by pakrym — APPROVED** [review]
Seems fine as a temporary solution

**Pull request merged**